website/azurerm.erb

```diff
@@ -1263,10 +1263,18 @@
                   <a href="/docs/providers/azurerm/r/sql_server.html">azurerm_sql_server</a>
                 </li>
 
+                <li>
+                  <a href="/docs/providers/azurerm/r/mssql_database.html">azurerm_mssql_database</a>
```

Comment:
It looks like this was already specified [here](https://github.com/terraform-providers/terraform-provider-azurerm/blob/91e17becf20a7816baa4e7c7ecadc8ea80819ea8/website/azurerm.erb#L1239). Do we want to remove that one as well?